src/styles/blocks/product-card.scss

```
@@ -0,0 +1,82 @@
+.product-card {
+  box-sizing: border-box;
+  display: flex;
+  width: 200px;
+  margin-top: 50px;
+  margin-right: 48px;
+  padding: 29px 14px 14px;
+  flex-wrap: wrap;
+  justify-content: center;
+  border: 1px solid #f3f3f3;
+
+  &:hover {
+    box-shadow: 0 2px 5px rgba(0, 0, 0, 0.07);
+  }
```

```suggestion
  &:hover {
    box-shadow: 0 2px 5px rgba(0, 0, 0, 0.07);
  }
```
Цього в принципі не потрібно було робити, але за ініціативу LIKE!